strawberryfields/io.py

```diff
@@ -244,6 +245,66 @@ def is_free_param(param):
     return prog
 
 
+def serialize_program(prog, eng=None):
+    """Converts a Strawberry Fields program into valid Strawberry Fields code.
+
+    Args:
+        prog (Program): the Strawberry Fields program
+        eng (Engine): the Strawberryfields engine
+
+    Returns:
+        str: the Strawberry Fields code, for constructing the program, as a string
+    """
+    code_seq = ["import strawberryfields as sf"]
+
+    operations = []
+    op_imports = []
+    for cmd in prog.circuit:
+        op_name = cmd.op.__class__.__name__
+        if op_name not in op_imports:
+            op_imports.append(op_name)
+
+        if prog.type == "tdm":
+            format_dict = {k: f"p[{k[1:]}]" for k in prog.parameters.keys()}
+            op = cmd.__str__().format(**format_dict)
+        else:
+            op = cmd.__str__()
+
+        operations.append(op)
+
+    code_seq.append("from strawberryfields.ops import " + ", ".join(op_imports) + "\n")
```

Comment:
Very minor: we could alternatively just do `from strawberryfields import ops`, and then prepend all operations with `ops.`. Not sure if this has any advantages over this approach though.

Reply:
Not sure what we'd prefer actually. I don't mind either way; I just chose this since I got the feeling that this would be the preferred way. The server actually has `from strawberryfields.ops import *`, but I'm not super-fond of that. I think either like this or, as you suggested, `from strawberryfields import ops`. I'm happy either way. :smile:

Reply:
Same, I actually have no preference! Feel free to choose which ever one you prefer.

The _only_ downside that I can think of: if the number of operations is large, the number of imports here will cause a very long line.

Reply:
I went with `from strawberryfields import ops`. :fireworks: